ethpm/validation.py

```diff
@@ -33,6 +33,16 @@ def validate_package_name(pkg_name: str) -> None:
         raise ValidationError("{0} is not a valid package name.".format(pkg_name))
 
 
+def validate_manifest_version(version: str) -> None:
+    """
+    Raise an exception if the version is not 2.
+    """
+    if not int(version) == 2:
```

Comment:
I don't think it should be cast to an integer here as this would allow for a non-integer literal `2` to pass validation.